fakeDataGenerator.js

```diff
@@ -9,7 +9,7 @@ const createFakeData = () => {
       `photos/photo-${i}.jpg`,
       faker.lorem.sentence(),
       faker.random.boolean(),
-      faker.random.number({ min: 1, max: 101, precision: 3 }),
+      Math.ceil((i % 101) + 1),
```

Comment:
So that means that every listing will have 6 or 7 photos. Do you want to consider listings that might have a lot more photos or maybe listings that have no photos?

Reply:
I think that when setting up a user to be a host, it's more than likely they require you to have "x" photos to display of the house, so I'm gonna leave it at 6 or 7 each. Thanks for the feedback!